src/core/server/base.js

```diff
@@ -465,7 +465,17 @@ function baseServer(
             sameSite: config.get('cookieSameSite'),
             secure: config.get('cookieSecure'),
           });
-          _log.debug('Cleared auth cookie');
+          // This is the Django session cookie, it needs to be removed to allow
+          // users to log in again successfully after account deletion.
+          // See: https://github.com/mozilla/addons-frontend/issues/9495
+          req.universalCookies.remove('sessionid', {
+            domain: config.get('cookieDomain'),
+            httpOnly: true,
+            path: '/',
+            sameSite: config.get('cookieSameSite'),
+            secure: config.get('cookieSecure'),
```

Comment:
Are the `options` filters on what cookies are removed?  If so, isn't `domain` (and name) sufficient to remove all cookies for AMO.  (i.e. what do we gain by filtering further to if `sameSite`, for example, was set the same as it is the current config)

Reply:
I was expecting the `remove` method to set a sensible value for `set-cookie` but it was a wrong assumption. We need to pass almost all cookie attributes to force FF and other browsers to actually remove the cookies. Otherwise, browsers will simply ignore the `set-cookie` header.

Reply:
oh, that's unexpected.